layers/+lang/python/packages.el

```diff
@@ -334,6 +334,9 @@
         "sR" 'spacemacs/python-shell-send-region-switch
         "sr" 'python-shell-send-region)
 
+      ;; we call python-indent-guess-indent-offset ourselves so set to nil to avoid duplicate work
```

Comment:
Replace this comment with something similar to:
```elisp
;; Set `python-indent-guess-indent-offset' to `nil' to prevent guessing `python-indent-offset'
```